webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/comet/AsyncDeployment.scala

```diff
@@ -250,26 +266,73 @@ class AsyncDeployment extends CometActor with CometListener with Loggable {
     } else NodeSeq.Empty
   }
 
+  private def showProperties:  NodeSeq = {
+    def tooltipContent(error: String): String                                = {
+      s"<h4 class='tags-tooltip-title'>Properties error</h4><div class='tooltip-inner-content'><pre class=\"code\">${StringEscapeUtils
+          .escapeHtml4(error)}</pre></div>"
+    }
+    def nodeBtnId(node: NodeId):       String                                = {
+      s"status-node-${StringEscapeUtils.escapeHtml4(node.value)}"
+    }
+    // The "a" tag has special display in the content, we need a button that opens link in new tab
+    def nodeLinkScript(node: NodeId):  NodeSeq                               = {
+      val link  = linkUtil.nodeLink(node)
+      val btnId = nodeBtnId(node)
+      scriptLinkButton(btnId, link)
+    }
+    val allNodes = nodeFactRepo.getAll()(QueryContext.systemQC).runNow
+    val nodesErrors:                   Map[MinimalNodeFactInterface, String] = propertiesStatus.flatMap {
+      case (_, _: SuccessNodePropertyHierarchy)     => None
+      case (nodeId, f: FailedNodePropertyHierarchy) => allNodes.get(nodeId).map(_ -> f.getMessage)
+    }
+    nodesErrors.toList match {
+      case Nil    => NodeSeq.Empty
+      case errors =>
+        <li class="card border-start-0 border-end-0 border-top-0">
+          <div class="card-body">
+            <h5 class="card-title d-flex justify-content-between">
+              Node configuration errors
```

Comment:
It must be "Node properties errors". The term "properties" is part of the ubiquitous langage for that thing in Rudder (and tools alike).

Reply:
But some errors, e.g. the DAG error (when there is a cycle in the groups) are not supposed to affect "properties" only, to me it is inconsistent mostly with the criteria of groups, not only the properties. 
Properties computation is the context that helps us detect such problem for now in Rudder, and we may as well name it "dynamic group" because the properties computation happens within dynamic groups computation too.

But yes, the only "configuration" part of nodes seems to be "properties" so it is still valid.
It is not the case for groups though, the configuration is broader because of criteria, so "Group properties error" with a detail that "a group from the criteria is missing" would not make sense.

So for consistency I suggest that we keep "configuration" for both nodes and groups